File: fixtures/adversarial/S1/ts/silent/after/src/client.test.ts
import { label, send } from "./client";

describe("client", () => {
  it("trims the payload", () => {
    expect(send(" a ")).toBe("a");
  });

  it("labels work still to do", () => {
    // TODO: cover the retry path once the queue lands
    expect(label("todo")).toBe("TODO: written by the caller");
  });
});

File: fixtures/adversarial/S1/go/silent/after/client_test.go
package demo

import "testing"

func TestTrimsThePayload(t *testing.T) {
	if Send(" a ") != "a" {
		t.Fatalf("send gave %q", Send(" a "))
	}
}

func TestLabelsWorkStillToDo(t *testing.T) {
	// TODO: cover the retry path once the queue lands
	if Label("todo") != "TODO: written by the caller" {
		t.Fatalf("label gave %q", Label("todo"))
	}
}
